news/str_date.rst

```diff
@@ -0,0 +1,12 @@
+**Added:**
+    * get_dates function that now works with objects with datetime objects
```

Comment:
I think you didn't add a get_dates function, you fixed it.... (see below)

Reply:
Fixed